pallets/passkey/src/mock.rs

```diff
@@ -1,23 +1,24 @@
 //! Mocks for the Time-release module.
-
-use super::*;
+use common_primitives::node::AccountId;
 use frame_support::{
 	construct_runtime,
 	traits::{ConstU32, Everything},
 };
 use sp_core::H256;
-use sp_runtime::{traits::IdentityLookup, BuildStorage};
+use sp_runtime::{
+	traits::{Convert, ConvertInto, IdentityLookup},
+	AccountId32, BuildStorage,
+};
 
 use crate as pallet_passkey;
 
-pub type AccountId = u128;
 impl frame_system::Config for Test {
 	type RuntimeOrigin = RuntimeOrigin;
 	type RuntimeCall = RuntimeCall;
 	type Nonce = u64;
 	type Hash = H256;
 	type Hashing = ::sp_runtime::traits::BlakeTwo256;
-	type AccountId = AccountId;
+	type AccountId = u64;
```

Comment:
Should we use `crate::msa::MessageSourceId` here instead? I'd prefer not to propagate `u64` in different places if it's all referencing the same conceptual type.